include/zenoh_macros.h

```diff
@@ -4,10 +4,8 @@
 
 #ifndef __cplusplus
 
-static inline z_moved_alloc_layout_t* z_alloc_layout_move(z_owned_alloc_layout_t* x) { return (z_moved_alloc_layout_t*)(x); }
```

Comment:
could you regenerate headers with unstable = ON ?

Reply:
Do we provide unstable headers by default now? Last time I touched it it was the other way round

Reply:
Checked, at least since 1.0.0 unstable was by default, fixed